services/graph/libs/common/src/config/config.service.ts

```diff
@@ -55,39 +70,31 @@ export class ConfigService {
     return this.nestConfigService.get<string>('PROVIDER_ACCESS_TOKEN');
   }
 
-  public getProviderId(): string {
+  public get providerId(): string {
     return this.nestConfigService.get<string>('PROVIDER_ID')!;
   }
 
-  public getQueueHighWater(): number {
+  public get queueHighWater(): number {
     return this.nestConfigService.get<number>('QUEUE_HIGH_WATER')!;
   }
 
-  public getApiPort(): number {
+  public get apiPort(): number {
     return this.nestConfigService.get<number>('API_PORT')!;
   }
 
-  public getReconnectionServiceRequired(): boolean {
+  public get reconnectionServiceRequired(): boolean {
     return this.nestConfigService.get<boolean>('RECONNECTION_SERVICE_REQUIRED')!;
   }
 
-  public getBlockchainScanIntervalMinutes(): number {
-    return this.nestConfigService.get<number>('BLOCKCHAIN_SCAN_INTERVAL_MINUTES')!;
-  }
-
-  public getRedisUrl(): URL {
-    return this.nestConfigService.get<URL>('REDIS_URL')!;
+  public get blockchainScanIntervalSeconds(): number {
+    return this.nestConfigService.get<number>('BLOCKCHAIN_SCAN_INTERVAL_SECONDS')!;
   }
 
-  public getFrequencyUrl(): URL {
-    return this.nestConfigService.get<URL>('FREQUENCY_URL')!;
-  }
-
-  public getGraphEnvironmentType(): keyof EnvironmentType {
+  public get graphEnvironmentType(): keyof EnvironmentType {
```

Comment:
Is this still needed?

Reply:
For now, until `graph-sdk` gets updated to use chain-retrieved schema IDs instead of an embedded config